src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -84,11 +84,15 @@
 			<ActionSeparator />
 
 			<ActionButton v-if="canLeaveConversation"
+				:close-after-click="true"
+				:disabled="item.hasCall"
```

Comment:
* So I create a conversation
* Invite the full company
* Start a call
* No one can leave it
* muhahaha

Maybe only when you are part of the call?

Reply:
I knew something was fishy... Couldn't be that easy.

To detect "is in call" need to bring in two mixins :-/

I'll see if I can still solve this differently and inhibit the dialog while the redirect is in progress.

Reply:
I can try moving this computed property to the store. Store getters are supposed to be like global computed properties and we've needed this one in enough places already.

Reply:
ok, as discussed this is a can of worms